README.md

```diff
@@ -80,13 +80,42 @@ If ok, response is 200:
 ```
 { message: 'Your offer has been sent!' }
 ```
+
 Some other errors can happen as well, unauthorized 401 if headers are missing:
+
 ```
 {"errors"=>["You need to sign in or sign up before continuing."]}
 ```
+
 Or 422 if you try a forbidden action or have bad params:
+
 ```
 { message: 'You cannot offer help on your own request!' }
 { message: 'Helper_id is already registered with this request' }
 { message: "Couldn't find Request with 'id'=blob" }
-```
\ No newline at end of file
+```
+
+#### PUT /offers
+
+Auth headers are required. Param :activity that can be declined or accepted as string.
+
+```
+{ offer: offer.id, message: 'offer is accepted' }
+{ offer: offer.id, message: 'offer is declined' }
+```
+
+if other activity response status 500 with message:
+
+```
+{ error_message: 'The activty is not valid' }
+```
+
+#### GET /offers/:id
+
+Auth headers are required. Offer :id in endpoint.
+
+```
+{ offer: offer.id, message: 'Your offer is pending' }
+{ offer: offer.id, message: 'Your offer has been accepted' }
+{ offer: offer.id, message: 'Your offer has been declined' }
+```
```

Comment:
GET /offers/:id should return the message that the helper sent off when he made the offer!

Reply:
Actually, when I look at the show action, `offer:` is not `offer.id` but the whole object. So then we have all information, including the offer message, that we need in there, which you also test for. Just update the readme 👍

Reply:
Gr8 spotted, updated